pkg/telemetry/license.go

```diff
@@ -0,0 +1,41 @@
+package telemetry
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+const (
+	LicenseEndpoint = "https://license.testkube.io/owner" //this is default but it can be set using ldflag -X github.com/kubeshop/testkube/pkg/telemetry.LicenseEndpoint=https://license.localhost
+)
+
+type EmailResponse struct {
+	Owner struct {
+		Email string `json:"email"`
+	} `json:"owner"`
+}
+type EmailRequest struct {
+	License string `json:"license"`
+}
+
+// GetEmail returns email
+func GetEmail(license string) string {
+	if LicenseEndpoint != "" {
+		payload := EmailRequest{License: license}
+		jsonPayload, _ := json.Marshal(payload)
+		resp, err := http.Post(LicenseEndpoint, "application/json", bytes.NewBuffer(jsonPayload))
+		if err != nil {
+			return ""
```

Comment:
do we need to log errors?

Reply:
it's for telemetry,and cli, we don't want for now